pyproject.toml

```
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["jupyter_packaging~=0.7.9", "jupyterlab>=3.0.0rc13,==3.*", "packaging", "setuptools>=40.8.0", "wheel"]
+requires = ["jupyter_packaging~=0.7.9", "jupyterlab>=3.0.0rc13,==3.*,<4", "packaging", "setuptools>=40.8.0", "wheel"]
```

That one is not needed because it is use only to build the package and Python requirement `>=3.0.0rc13,==3.*` is equivalent to `>=3.0.0rc13` **and** `==3.*` (see [PEP](https://peps.python.org/pep-0440/#version-specifiers))

```suggestion
requires = ["jupyter_packaging~=0.7.9", "jupyterlab>=3.0.0rc13,==3.*", "packaging", "setuptools>=40.8.0", "wheel"]
```